setup.py

```diff
@@ -452,6 +452,7 @@ def build_extensions(self):
 	packages=find_packages("Lib"),
 	include_package_data=True,
 	data_files=find_data_files(),
+	zip_safe=False,  # So mypy can find typing information.
```

Comment:
that's bad.. don't they use the importlib.metadata api?

Reply:
I don't know. https://mypy.readthedocs.io/en/stable/installed_packages.html?highlight=zip_safe#making-pep-561-compatible-packages

Reply:
https://github.com/python/mypy/issues/8802

Reply:
you can remove it, we don't support legacy `setup.py install`, see jaraco's comment in the above linked issue